<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SW_TASK_Dialog</class>
 <widget class="QWidget" name="SW_TASK_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressure Nulling Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_FT_NULL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F/T Nulling Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_PRES_CALIB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pressure calibration Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MOVE">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_1000">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_PWM_ON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PWM ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_PWM_OFF">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PWM OFF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_PLUS_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_PLUS_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_PLUS_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_PLUS_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_PLUS_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SHOW">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SHOW</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_MINUS_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_MINUS_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_MINUS_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_MINUS_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_VALV_GAIN_MINUS_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>531</width>
     <height>471</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="LE_PWM_SINE_DUTY">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PWM_SINE">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_PWM_SINE_FREQ">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
